lib/fluent/plugin/out_lm.rb

```diff
@@ -194,6 +194,9 @@ def send_batch(events)
       resp = http.request(request)
       if @debug || (!resp.kind_of? Net::HTTPSuccess)
```

Comment:
@shawnj , We always want to log a 207 response, by above logic, it seems like we will log this only when debug is enabled. 
We can have the condition like : 
`if @debug || (!resp.kind_of? Net::HTTPSuccess || resp.kind_of? Net::HTTPMultiStatus )` and remove the next if block.

Reply:
@siddharthck As you can see the `response.body` is not part of the log.  That was part of the original PR to include that.